{{cookiecutter.plugin_name}}/setup.cfg

```diff
@@ -67,6 +69,8 @@ install_requires =
     napari-plugin-engine>=0.1.4
     numpy
 
+[options.packages.find]
+where = src
```

Comment:
are both `package_dir=src` and `[options.packages.find] where=src` required? Why?

Reply:
Base on my knowledge yes. `package_dir=src` told distutils relative root path https://docs.python.org/3/distutils/setupscript.html#listing-whole-packages

`[options.packages.find] where=src` is option for `find()` function. it could be removed by hardcode `packages` variable value.